torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -378,43 +379,44 @@ class TensorExprFuser {
     }
   }
 
+  value_list sortReverseTopological(ArrayRef<Value*> inputs, Block* b) {
+    value_list result;
+    for (auto i : inputs) {
+      if (i->node()->owningBlock() == b) {
+        result.push_back(i);
+      }
+    }
+    // Sort in reverse topological order
+    std::sort(result.begin(), result.end(), [&](Value* a, Value* b) {
+      return a->node()->isAfter(b->node());
+    });
+    return result;
+  }
+
   // Create a fusion group starting from the node N.
   // We then try to pull inputs into the fusion group and repeat that process
   // until there is nothing we can pull in.
-  Node* createFusionGroup(Node* n) {
-    // Queue of the nodes we should consider for merging into the fusion groups
-    // (those nodes are usually inputs of the fusion group).
-    // We use an ordered set here to visit them in the right order: the fusion
-    // group is closer to the end of the block and we are trying to pull later
-    // nodes first.
-    // NB: the order in the list in theory could stale if we move nodes around.
-    // However, this should only happen to the nodes we could not fuse, and
-    // hence it should not be a problem.
-    std::set<Node*, nodesComparator> queue;
-    std::unordered_set<Node*> visited_nodes;
-
+  std::pair<graph_node_list::iterator, bool> createFusionGroup(Node* n) {
     Node* fusion_group = n;
+    bool changed = false;
     if (min_group_size_ == 1) {
       fusion_group = getOrCreateTensorExprSubgraph(n);
+      changed = true;
     }
 
-    updateQueue(fusion_group, queue, visited_nodes);
```

Comment:
You can probably garbage collect the updateQueue function since it appears no longer used.